thoth/storages/graph/postgres.py

```diff
@@ -4284,8 +4284,8 @@ def sync_analysis_result(self, document: dict) -> None:
         environment_type = environment_type.upper()
         origin = document["metadata"]["arguments"]["thoth-package-extract"]["metadata"].get("origin")
         environment_name = document["metadata"]["arguments"]["extract-image"]["image"]
-        os_name = document["result"]["operating-system"]["name"]
-        os_version = document["result"]["operating-system"]["version_id"]
+        os_name = document["result"].get("operating-system", {}).get("name")
+        os_version = document["result"].get("operating-system", {}).get("version_id")
```

Comment:
The exception looks suspicious, these values should be always present:

https://github.com/thoth-station/package-extract/blob/54080efc724c0b353c85024f0901a6485a43b040/thoth/package_extract/image.py#L543-L545

Reply:
@bissenbay could you verify package-extract run (how was it triggered and what results it produced). package-extract might be also in an old version in deployments.